capstone-backend/src/main/java/com/google/sps/servlets/Utility.java

```diff
@@ -36,11 +36,14 @@ public static Firestore getFirestoreDb() throws IOException {
       return FirestoreClient.getFirestore();
     }
 
-    GoogleCredentials credentials = GoogleCredentials.getApplicationDefault();
+    FileInputStream serviceAccount = new FileInputStream(
```

Comment:
How do these new credentials work?

Reply:
It contains information about our Firebase project. You can check it out on the Pod Meetings Google Drive Folder.

Reply:
Cool!